src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SentReferralDTO.kt

```diff
@@ -12,6 +12,8 @@ class SentReferralDTO(
   val assignedTo: AuthUserDTO?,
   val referral: DraftReferralDTO,
   val actionPlanId: UUID?,
+  val currentActionPlanId: UUID?,
```

Comment:
Essentially a re-name of `actionPlanId`. Leaving the old field here temporarily until the front end has been updated to `currentActionPlanId`

Reply:
just a heads up - we should never remove the old field, to avoid potential backwards incompatibility issues.